### PR TITLE
feat(LAB-3177): Add possibility to filter on deleted projects for projects() and count_projects()

### PR DESCRIPTION
**Pull request by stchabi**

### CODE REVIEW

File: src/kili/presentation/client/project.py [hunk 1]
```diff
@@ -426,5 +431,6 @@ def count_projects(
                 updated_at_gte=updated_at_gte,
                 updated_at_lte=updated_at_lte,
                 archived=archived,
+                deleted=deleted,
```
Comment (baptiste-olivier) [inline src/kili/presentation/client/project.py:434]: Add a quick description in the documentation above to explain what this field is doing

File: src/kili/presentation/client/project.py [hunk 2]
```diff
@@ -265,6 +268,7 @@ def projects(
                 created_at_lte=None,
                 organization_id=organization_id,
                 tag_ids=tag_ids,
+                deleted=deleted,
```
Comment (baptiste-olivier) [inline src/kili/presentation/client/project.py:271]: Add a quick description in the documentation above to explain what this field is doing

Comment (stchabi) [reply src/kili/presentation/client/project.py:272]: Done ! ✅

File: src/kili/presentation/client/project.py (hunk 1 above)
Comment (stchabi) [reply src/kili/presentation/client/project.py:436]: Done ! ✅

File: src/kili/presentation/client/project.py [hunk 3]
```diff
@@ -235,6 +238,7 @@ def projects(
             skip: Number of projects to skip (they are ordered by their creation).
             disable_tqdm: If `True`, the progress bar will be disabled.
             as_generator: If `True`, a generator on the projects is returned.
+            deleted: If `True`, all projects are returned (including deleted ones), if `False` only not deleted ones are returned.
```
[the remark this answers is not in the document]
Comment (florianlega) [reply src/kili/presentation/client/project.py:241]: Maybe explain the default behavior instead ?

**Review by baptiste-olivier — APPROVED** [review]

**Review by florianlega — APPROVED** [review]

**Pull request merged**